src/libs/scene_utils/client/iottwinmaker.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 2022
+// SPDX-License-Identifier: Apache-2.0
+
+import { IoTTwinMaker } from 'aws-sdk';
+import {
+  CreateSceneRequest,
+  GetWorkspaceRequest,
+  EntitySummaries,
+  ListEntitiesFilters,
+  ListEntitiesRequest,
+  DeleteSceneRequest,
+} from 'aws-sdk/clients/iottwinmaker';
+
+export class IotTwinMakerClient {
+  private twinmakerClient: IoTTwinMaker;
+
+  constructor() {
+    this.twinmakerClient = new IoTTwinMaker();
```

Comment:
may be useful to embed a small user-agent header to indicate this request is coming from Scene SDK so we can help trace issues for customers

Reply:
Great idea! I'll add that and verify in our service logs

Reply:
Added the user agent header: 
`"Node.js/{Node version} ({OS}/{Arch}) {SceneUtils package name}/{SceneUtils package version}"`